packages/web-components/src/stories/time-region.stories.mdx

```diff
@@ -9,9 +9,9 @@ args.status.defaultValue = 'standby'
 export const StyledDiv = styled.div`
     position: relative;
     margin: 1rem 0;
-    border-left: 20px solid var(--status-symbol-color-fill-serious);
+    border-left: 20px solid var(--color-status-serious);
     background: white;
-    color: var(--status-symbol-color-fill-serious);
+    color: black;
```

Comment:
whats up with these story changes? is the branch just stale from main?

Reply:
Prob some leftover crap from testing that I forgot to remove. Will revert that